hw/drivers/sensors/lis2dh12/src/lis2dh12.c

```diff
@@ -860,6 +1089,8 @@ lis2dh12_config(struct lis2dh12 *lis2dh12, struct lis2dh12_cfg *cfg)
 
     lis2dh12->cfg.lc_s_mask = cfg->lc_s_mask;
 
+    os_time_delay((OS_TICKS_PER_SEC * 110)/1000);
```

Comment:
Sorry, I was trying different configs in a desperate attempt to get it working. This delay is needed for any initial interrupts that might occur but for the polling config it really doesn't matter. Removing it.